plugins/chat/assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -142,12 +208,17 @@ ChatChannel.reopenClass({
     this._initUserModels(args);
     this._initUserMembership(args);
 
-    args.chatableType = args.chatable_type;
-    args.membershipsCount = args.memberships_count;
+    this._remapKey(args, "chatable_type", "chatableType");
+    this._remapKey(args, "memberships_count", "membershipsCount");
+    this._remapKey(args, "last_message_sent_at", "lastMessageSentAt");
 
     return this._super(args);
   },
 
+  _remapKey(obj, oldKey, newKey) {
```

Comment:
This could be useful to extract for other uses. Also, why not use this for `set details(details)` above?

Reply:
It will be all gone when I move chat-channel model to the same pattern used in chat-message